.github/workflows/update-foundry-prove.yml

```
@@ -0,0 +1,27 @@
+name: 'Update Foundry prove'
+on: push
+jobs:
+  update-output:
+    name: 'Update Foundry expected output'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out code'
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
```

```suggestion
      - name: 'Check out code'
        uses: actions/checkout@v3
        with:
          submodules: recursive
          token: ${{ secrets.JENKINS_GITHUB_PAT }}
```

This is to make sure that `git push` will work.